Factory Environmental Design

SirTangoGibus opened the pull request:
Test it, give it an art pass, go to town with it, you're probably reading this at 2 AM anyway. How's life bub?

SirTangoGibus committed:
Factory Environmental Design and Assets

feat: Added a shitton of assets
feat: Factory environmental design wip

SirTangoGibus committed:
Further work on factory prefab

feat: Walls and ceiling done

SirTangoGibus committed:
Factory Complete

feat: Added props too

SirTangoGibus committed:
Cream material

feat: Forgot if i used it or not in the factory scene so pushing it lol

IllegaaI reviewed (changes requested):
So, I think that the factory looks good and does not have any real visual errors or any bugs not seen before the changes. But, I did find that the sun shines right through the side walls and makes everything looks it does not have shadows. Besides that, there is also the issue of not having implemented the plug and socket and there not being realized space for that at the moment. So, after those small fixes this branch can be merged.

SirTangoGibus committed:
Cave Level

feat: added walls

SirTangoGibus committed:
Merge remote-tracking branch 'origin/main' into FactoryandCaveENVD

SirTangoGibus committed:
Cave Level Environmental Design

Feat: Added lighting, props etc to cave

MGHMSJ5 committed:
Cave fixes

fix(cave prefab): added repair particle to the water pump, referenced the right UI for the waterpump and pipe (in the repair script), + added collider to the water pump

MGHMSJ5 committed:
Cave & Planet light fix

fix(directional lights): made it so that the teleport points to and form the cave will disable and enable the right directional light

IllegaaI committed:
Added Basic Materials, Updated Moving Platforms & Factory  Plug

feat & fix(FactoryLevel, CaveLevel, Platform & Kettler22T): added basic materials, 3 moving platforms variants, implemented these into the factory and cave with some other changes to the panels, and  put in the plug and socket with its accompanying code

SirTangoGibus committed:
Merge remote-tracking branch 'origin/main' into FactoryandCaveENVD

SirTangoGibus committed:
Post processing and Camera movement

feat: Post processing, changes to cinemachine movement (camera radius, camera smoothing and preserves camera dsitance for better camera control).

IllegaaI committed:
Factory & Cave Platform Updates

fix(platforms): changed some of the platforms to be more platform-able

IllegaaI reviewed (approved):
Checked it all out, it now is mergeable in my opinion!
